src/pages/HomePage.js

```diff
@@ -1,13 +1,66 @@
 import React from "react";
-import { useSelector } from "react-redux";
+import "../style/App.scss";
+import "../style/HomePage.scss";
+import ExampleComponent from "react-rounded-image";
+import defaultProfileImg from "../assets/profilePicture.png";
+import { FormattedMessage } from "react-intl";
+import { Map, TileLayer } from "react-leaflet";
+import { mapConstants } from "../constants/map.constants";
+import menuIcon from "../assets/icons/menu_icon.png";
+import smilies from "../assets/smilies.png";
```

Comment:
```suggestion
import { FormattedMessage } from "react-intl";
import { Map, TileLayer } from "react-leaflet";
import ExampleComponent from "react-rounded-image";

import defaultProfileImg from "../assets/profilePicture.png";
import { mapConstants } from "../constants/map.constants";
import menuIcon from "../assets/icons/menu_icon.png";
import smilies from "../assets/smilies.png";
import "../style/App.scss";
import "../style/HomePage.scss";
```

 I usually separate with a new line the installed packages from my own components